src/resources/projects/ProjectList.tsx

```diff
@@ -11,7 +11,7 @@ import {
   TopToolbar
 } from 'react-admin'
 
-const filters = [<SearchInput source='name' key='name' alwaysOn />]
+const filters = [<SearchInput source='q' key='q' alwaysOn />]
```

Comment:
Hello @TahaKhanAbdalli - would just this change have fixed the issue?

Do we need to modify the data provider?

Reply:
Yes, It by default searches all the properties but in the issue filter was specified on the name field therefore I tried to add only on the name field

Reply:
Aah yes.  _For-now_, I'm tempted to stick with the `q` wildcard search.